pmd-java/src/main/java/net/sourceforge/pmd/lang/java/rule/design/FieldDeclarationsShouldBeAtStartOfClassRule.java

```diff
@@ -40,6 +40,8 @@
             "Ignore Enum Declarations that precede fields.", true, 1.0f);
     private BooleanProperty ignoreAnonymousClassDeclarations = new BooleanProperty("ignoreAnonymousClassDeclarations",
             "Ignore Field Declarations, that are initialized with anonymous class declarations", true, 2.0f);
+    private BooleanProperty ignoreInterfaceDeclarations = new BooleanProperty("ignoreInterfaceDeclarations",
```

Comment:
While writing the unit test, I figured out, that one piece is missing. The properties need to be declared for the rule. This usually happens in the constructor like this:

    definePropertyDescriptor(ignoreInterfaceDeclarations);

I'll add this. It's always good to have unit tests :)